ovos_persona_server/gemini.py

```
@@ -0,0 +1,125 @@
+        SSE stream of GeminiResponse JSON objects.
+    """
+    messages = _normalise_messages(request)
+
+    async def _stream() -> AsyncGenerator[str, None]:
+        """Yield SSE events with GeminiResponse chunks."""
+        try:
+            for chunk in persona.stream(messages):
+                if chunk:
+                    resp = _build_response(chunk, model_id)
+                    yield f"data: {json.dumps(resp.model_dump())}\n\n"
+        except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)